ee/clickhouse/views/test/__snapshots__/test_clickhouse_retention.ambr

```diff
@@ -1,3 +1,212 @@
+# name: RetentionTests.test_retention_aggregation_by_distinct_id_and_retrieve_people
+  '
+  /* request:api_projects_(?P<parent_lookup_team_id>[^_.]+)_insights_retention_?$ (ClickhouseInsightsViewSet) */ WITH actor_query AS
+    (WITH 'day' as period,
+          NULL as breakdown_values_filter,
+          NULL as selected_interval,
+          returning_event_query as
+       (SELECT toStartOfDay(e.timestamp) AS event_date,
+               e.uuid AS uuid,
+               e.event AS event,
+               e.distinct_id as target
```

Comment:
distinct_id is target for regular query, but pdi.person_id is target when querying persons for the same query.